terminy.html

```diff
@@ -28,6 +28,8 @@
         <li class="mr-3">
           <a class="inline-block border border-white rounded py-1 px-3 bg-blue-500 text-white
                     hover:text-blue-500 hover:border-gray-200 hover:bg-gray-200" href="index.html">O nás</a>
+          <a class="inline-block border border-white rounded py-1 px-3 bg-blue-500 text-white
+              hover:text-blue-500 hover:border-gray-200 hover:bg-gray-200" href="ical.html">iCal</a>
```

Comment:
Nevieme tomu vymysliet iny nazov ? (kalendar sa zrovna nehodi a ical podla mna malokto pozna :D )

Reply:
veď toto :slightly_smiling_face: tiež absolútne neviem, čo tam dať, aby to nebolo dlhé a zároveň aby to človek pochopil